README.md

```diff
@@ -15,75 +15,72 @@
-| [Hops Build Config](https://github.com/xing/hops/tree/master/packages/build-config) | [![npm](https://img.shields.io/npm/v/hops-build-config.svg)](https://www.npmjs.com/package/hops-build-config) |
-| [Hops Jest Preset](https://github.com/xing/hops/tree/master/packages/jest-preset)   | [![npm](https://img.shields.io/npm/v/jest-preset-hops.svg)](https://www.npmjs.com/package/jest-preset-hops)   |
+#### Why a framework and not just a few code snippets describing best practices?
 
-The following additional packages are rather low-level and you'll probably never need to install and use them directly.
+There are two main reasons. Firstly, snippets out-date. Hops, however, is committed to best practices and evolve with them. Secondly, Hops is organized in modules that need to run both on the client and on the server. Keeping a track of both sides and at the same time maintaining an in sync version of a client side and server side snippet per module can be really tedious. Using a framework like Hops takes away that pain.
 
-| Package                                                                         | NPM                                                                                                       |
-| ------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
-| [Hops Server](https://github.com/xing/hops/tree/master/packages/server)         | [![npm](https://img.shields.io/npm/v/hops-server.svg)](https://www.npmjs.com/package/hops-server)         |
-| [Hops Middleware](https://github.com/xing/hops/tree/master/packages/middleware) | [![npm](https://img.shields.io/npm/v/hops-middleware.svg)](https://www.npmjs.com/package/hops-middleware) |
-| [Hops Renderer](https://github.com/xing/hops/tree/master/packages/renderer)     | [![npm](https://img.shields.io/npm/v/hops-renderer.svg)](https://www.npmjs.com/package/hops-renderer)     |
+#### Why does Hops not support "eject"?
+
+[Create React App](https://github.com/facebook/create-react-app) offers you the option to "eject". This means dumbing Webpack configuration files and scripts, leaving you with a stand-alone project. We Hops people do not think, this is a good idea. Those files still force their philosophy on you while at the same time they are hard to understand. We thus do not believe this to be a practical way overcome vendor lock-in. Hops instead allows you to customize either by configuration or adding modules.
+
+#### Can we trust in Hops to be around for the forseeable future?
+
+Just as you might put trust into React as Facebook has committed itself to it and eats its own dog food, Hops is also backed by and originated from one of the leading contact social networking sites.
```

Comment:
How about "Hops is being backed and used by a European professional networking site?"